spec/types/go/primitives/lean_helix_primitives.mb.go

```diff
@@ -1,10 +1,4 @@
-// Copyright 2019 the lean-helix-go authors
-// This file is part of the lean-helix-go library in the Orbs project.
-//
-// This source code is licensed under the MIT license found in the LICENSE file in the root directory of this source tree.
-// The above notice should be included in all copies or substantial portions of the software.
-
-// AUTO GENERATED FILE (by membufc proto compiler v0.0.21)
+// AUTO GENERATED FILE (by membufc proto compiler v0.0.32)
```

Comment:
we lost the header. headers were added uniformly after the source had already been generated. maybe best to add it here manually - but this is a project wide concern...

Reply:
auto generated code should also auto generate the notice ... or it will be very annoying :)